net/tinc/files/tinc.init

```diff
@@ -171,7 +217,7 @@ start_instance() {
 	append_params "$s" logfile debug
 
 	SERVICE_PID_FILE="/var/run/tinc.$s.pid"
-	service_start $BIN -c "$TMP_TINC/$s" -n $s $ARGS --pidfile="$SERVICE_PID_FILE"
+	service_start $BIN -c "$TMP_TINC/$s" $ARGS --pidfile="$SERVICE_PID_FILE"
```

Comment:
Why you are dropping `-n $s`. You plan on dropping support for different network configurations ?
Are you aware of:
https://github.com/openwrt/packages/issues/9914#issuecomment-702972496 ?

Reply:
i dont plan to remove for different network configurations .
When you start with `-c`  and `-n`  you have this warning message with tinc 1.1 
```
Both netname and configuration directory given, using the latter...
```

Reply:
by removing -n , i remove only the warning this will not affect multiple network configuration